### ISSUE TITLE
Add an animation to refresh and setting Icon.

### ISSUE THREAD

**Issue by alanbosco003** [alanbosco003/Boxfile#5]
Add a simple animation to the refresh and setting Icon. Right now we are just rotating these icons but it will be much better if we are able to rotate it in an animated way so that the use can see the icon is getting rotated.

![Screenshot_2023-04-10-20-37-02-15_da10d9fb4558d7f55ea05b05124829f4](https://user-images.githubusercontent.com/67573902/230930218-6bc02394-c6d3-4a1b-a1f4-c6e97cbeb276.jpg)
![Screenshot_2023-04-10-20-37-19-94_da10d9fb4558d7f55ea05b05124829f4](https://user-images.githubusercontent.com/67573902/230930231-b7a49588-5f2d-4ef4-a540-9d1710b0d781.jpg)

**Comment by Naainz** [comment]
Can you please help me with the file name, and what the name of the refresh icon is in that menu?

**Comment by alanbosco003** [comment]
Yes sure @Naainz, the refresh button is inside filespage.dart ( line number 57 ) else you can search for this code 
`const Icon(Icons.refresh, size: 28, color: Color(Colorz.oranch)),` the settings button is inside editfiles.dart page (line number 381 ) else you can search for code `icon: const Icon(Icons.settings,`
Thanks

**Comment by gantanikhilraj** [comment]
Hello @alanbosco003 , I can work on this. Assign it to me.
